README.brazilian-portuguese.md

```diff
@@ -1166,6 +1166,6 @@ Nossos colaboradores são membros que estão contribuindo com o repositório em
 
 ## Thank You Notes
 
-We appreciate any contribution, from a single word fix to a new best practice. View our contributors and [contributing documentation here!](CONTRIBUTORS.md)
+We appreciate any contribution, from a single word fix to a new best practice. View our contributors and [contributing documentation here!](./README.md#contributors-)
```

Comment:
These changes (and similar in the other README files) happens because the CONTRIBUTORS page was merged to the main README.md file so instead I linked to the contribution section.

The leading `-` is because there is an emoji at the end